src/api/component-repository/components.js

```diff
@@ -108,6 +109,7 @@ export const getECRComponentInstallPlan = code => (
     domain: '/digital-exchange',
     method: METHODS.GET,
     mockResponse: COMPONENT_INSTALL_PLAN,
+    // mockResponse: FAILED_INSTALL_PLAN,
```

Comment:
should we delete this?

Reply:
yeah.. I thought about keeping it to know we have a FAILED mock response too, but I agree with you now, am going to remove it.